tests/test_meshtastic_connection.py

```
@@ -564,6 +564,133 @@ def _configure_ble(self) -> str:
+                timeout: int | None = None,
+                auto_reconnect: bool = False,
+            ) -> None:
+                del noProto, debugOut, noNodes, timeout
+                self.address = address
+                self.auto_reconnect = auto_reconnect
+
+            def connect(self) -> None:
+                return None
+
+        class ImmediateExecutor:
+            def submit(self, fn, *args):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `submit`

(ANN202)